src/eql.erl

```diff
@@ -63,3 +73,24 @@ get_query(Name, Proplist) ->
         {Name, Value} -> {ok, Value};
         false -> undefined
     end.
+
+%% internal functions
+
+-spec file_to_namespace(file:filename_all()) -> atom().
+file_to_namespace(File) ->
+    list_to_atom(filename:basename(File, ".sql")).
+
+maybe_flush(Tab, File, Opts) ->
+    UseNamespace = proplists:get_value(namespace, Opts, false),
+    case proplists:get_value(flush, Opts, false) of
+        true ->
+            case UseNamespace of
```

Comment:
`case proplists:get_value(namespace, Opts, false) of` :)

Reply:
Heh, yea, I'll fix that, earlier version used `UseNamespace variable twice for some reason so it made sense to resolve once :).